test/units/modules/cloud/spotinst/test_spotinst_aws_elastigroup.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import sys
+from mock import MagicMock
+sys.modules['spotinst_sdk'] = MagicMock()
```

Comment:
Use a pytest fixture to modify sys.modules so this doesn't pollute other unit tests.

For an example of how to do this, take a look at https://github.com/ansible/ansible/pull/52012